pkg/factory/fake-factory.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package factory
+
+import (
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/kubectl/pkg/cmd/util"
+	"sigs.k8s.io/cli-utils/pkg/inventory"
+	"sigs.k8s.io/cli-utils/pkg/object"
+)
+
+type FakeInventoryFactory struct {
+	factory util.Factory
+	objs    []object.ObjMetadata
+}
```

Comment:
To help document the interface this implements, can we add 

```
var _ whatever.NameOfInterface = (*FakeInventoryFactory)(nil)
```

Reply:
Done.